### PR TITLE
Add FreeBSD armv6 and armv7 targets

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by MikaelUrankar** [comment]
Anyone?

**Comment by lzutao** [comment]
cc @Centril

**Comment by nagisa** [comment]
@bors r+

LGTM, thanks!

**Comment by Centril** [comment]
Failed in rollup (https://github.com/rust-lang/rust/pull/58412#issuecomment-463013825) due to 2018 changes,  @bors r-

File: src/librustc_target/spec/armv6_unknown_freebsd.rs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+use spec::{LinkerFlavor, Target, TargetOptions, TargetResult};
```
Comment (Centril) [inline src/librustc_target/spec/armv6_unknown_freebsd.rs]:
```suggestion
use crate::spec::{LinkerFlavor, Target, TargetOptions, TargetResult};
```

File: src/librustc_target/spec/armv7_unknown_freebsd.rs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+use spec::{LinkerFlavor, Target, TargetOptions, TargetResult};
```
Comment (Centril) [inline src/librustc_target/spec/armv7_unknown_freebsd.rs]:
```suggestion
use crate::spec::{LinkerFlavor, Target, TargetOptions, TargetResult};
```

**Comment by MikaelUrankar** [comment]
Is it ok now?

**Pull request merged**